README.md

```diff
@@ -22,7 +22,7 @@ Tiling Window Management for Windows.
   <a href="https://notado.app/feeds/jado/software-development">
     <img alt="Notado Feed" src="https://img.shields.io/badge/Notado-Subscribe-informational">
   </a>
-  <a href="https://www.youtube.com/channel/UCeai3-do-9O4MNy9_xjO6mg?sub_confirmation=1">
+  <a href="https://www.youtube.com/channel/@LGUG2Z?sub_confirmation=1">
```

Comment:
![image](https://github.com/LGUG2Z/komorebi/assets/13164844/98c031a5-b2b3-411e-a75d-7adf77eb14f3)

These need to keep the channel ID